Alteracoes neri

GabrielForner opened the pull request:
Implementado tela da listagem do estoque

GabrielForner committed:
implementado datamodule do estoque

implementado datamodule do estoque

GabrielForner committed:
Criado método com sql para buscar dados na tabela estoque

Criado método com sql para buscar dados na tabela estoque

GabrielForner committed:
Criada tela da listagem do estoque

Criada tela da listagem do estoque

GabrielForner committed:
Criada tela da listagem do estoque e métodos de busca

Criada tela da listagem do estoque e métodos de busca, preenchimento do grid

GabrielForner committed:
criados controles de edição no grid

criados controles de edição no grid

isaquevoss reviewed (commented):
Implementações ok, porém não está limitando a quantidade de caracteres e ao adicionar mais que 120 está apresentando mensagens de erro ao usuário.

GabrielForner committed:
Limitado edit da busca do produto

Limitado edit da busca do produto para 120 caracteres, e criado método para busca do produto no onChange do edit

GabrielForner commented:
- Adicionado MaxLength no Edit de busca
- Criado método para busca do produto ao digitar qualquer informação
- Removido botão de Buscar

GabrielForner committed:
Merge branch 'master' into alteracoes-neri

isaquevoss reviewed (approved):
Show neri, tudo ok
